ckanext/dcatapchharvest/harvesters.py

```diff
@@ -16,6 +16,7 @@ class SwissDCATRDFHarvester(DCATRDFHarvester):
     p.implements(IDCATRDFHarvester, inherit=True)
 
     harvest_job = None
+    current_page_url = None
```

Comment:
Added this because the new class attribute should be defined before we update it in `before_download()`. https://github.com/opendata-swiss/ckanext-dcatapchharvest/pull/57/files#diff-ed0eb3c194d5700090b40c0a288626fb4b4936ebd741f9c2fcb2581ec6ddf83aR60